rasa/core/test.py

```diff
@@ -7,7 +7,9 @@
 
 from rasa import telemetry
 from rasa.core.policies.policy import PolicyPrediction
+from rasa.nlu.test import EntityEvaluationResult, evaluate_entities
```

Comment:
should these be moved to shared?

Reply:
Why? I don't see any big reason of moving it to `shared`.